futures-channel/src/oneshot.rs

```diff
@@ -377,8 +377,16 @@ impl<T> Drop for Sender<T> {
 
 /// Error returned from a [`Receiver`](Receiver) when the corresponding
 /// [`Sender`](Sender) is dropped.
-#[derive(Clone, Copy, PartialEq, Eq, Debug)]
```

Comment:
I actually like this one as-is because it allows you to easily write it as a pattern-- e.g.

```rust
match recv.await {
    Ok(x) => ...,
    Err(Canceled) => ...
}
```

I can't really imagine what else we'd want to add to it in the future.

Reply:
Okay, these are in util/channel, so we can change them if we need them.

Reply:
FWIW, I tend to write it like:

```rust
match rx.await {
    Err(_canceled) => (),
}
```

Reply:
@seanmonstar That doesn't actually assert that cancelled is what happened there, though-- you could be silencing a real error by accident.